feagen/tools/feagen_runner.py

```diff
@@ -30,7 +28,7 @@ def feagen_run_with_configs(global_config, bundle_config, dag_output_path=None):
     bundle_path = join(global_config['data_bundles_dir'],
                        bundle_config['name'] + '.h5')
     bundle_data(bundle_config['structure'],
-                global_data_hdf_path=global_config['global_data_hdf_path'],
+                global_data_hdf_path=bundle_config['global_data_hdf_path'],
```

Comment:
You didn't understand. If you want to set the global_data_hdf_path for a bundle, you should also set it in the generator_kwargs, so it will override the global config.